.travis.yml

```diff
@@ -1,7 +1,25 @@
 language: dart
-dart:
-  - stable
-script:
-  - pub run dart_dev format --check
-  - pub run dart_dev analyze
-  - pub run dart_dev test
+sudo: required
+
+addons:
+  chrome: stable
+
+jobs:
+  include:
+    - dart: 2.4.0
+      name: "SDK: 2.4.0"
+      script:
+        - pub run dart_dev analyze
+        - pub run dart_dev test --release
```

Comment:
Since we only run the unit tests on `vm`, we probably don't need to run them with the `--release` flag.

Reply:
why not? is there no difference between them for `vm`?

Reply:
Yeah, it's meaningless in the context of the vm.